app/uk/gov/hmrc/fileupload/controllers/envelope.scala

```diff
@@ -60,45 +60,40 @@ object EnvelopeReport {
 case class CreateEnvelopeRequest(callbackUrl: Option[String] = None,
                                  expiryDate: Option[DateTime] = None,
                                  metadata: Option[JsObject] = None,
-                                 constraints: Option[EnvelopeConstraintsUserO] = None)
+                                 constraints: Option[EnvelopeConstraintsUserO] = Some(EnvelopeConstraintsUserO()))
```

Comment:
Option[String] in has not validated type.
Our API gives strict format on that String. This syntax validation have to be done on the controller or serialization layer.